src/Elastic.Apm/Logging/LogValuesFormatter.cs

```diff
@@ -183,11 +183,11 @@ public LogValues GetState(object[] values)
 			var args = new KeyValuePair<string, object>[values.Length + offset];
 			args[0] = new KeyValuePair<string, object>("{OriginalFormat}", OriginalFormat);
 			if (_scope != null)
-				args[1] = new KeyValuePair<string, object>("{Scope}", _scope);
+				args[1] = new KeyValuePair<string, object>("Scope", _scope);
 
 			for (int i = 0, j = _scope != null ? 1 : 0; j < ValueNames.Count; i++, j++)
```

Comment:
`j = offset - 1` and adding the condition to the for `j < ValueNames.Count && i < values.Length` reads slightly better to me. 

Unrelated to this PR though feel free to ignore 😄